src/config/logger.ts

```diff
@@ -0,0 +1,49 @@
+import { resolve } from 'path'
+import { createLogger, format, transports } from 'winston'
+
+const { combine, timestamp, label, printf } = format
+
+const myFormat = printf(({ level, message, label, timestamp }) => {
+  return `${timestamp} [${label}] ${level}: ${message}`
+})
+
+const { NODE_ENV } = process.env
```

Comment:
extract all vars in config.js.
use config js in application only.
```suggestion
const { NODE_ENV } = config.ENV
```